metakernel/_metakernel.py

```diff
@@ -79,6 +82,17 @@ class MetaKernel(Kernel):
     }
     meta_kernel = None
 
+    @classmethod
+    def run_as_main(cls):
+        """Launch or install a metakernel.
+
+        Modules implementing a metakernel subclass can use the following lines:
+
+            if __name__ == '__main__':
+                MetaKernelSubclass.run_as_main()
+        """
+        _MetaKernelApp.launch_instance(kernel_class=cls)
```

Comment:
I needed to change this to `IPKernelApp.launch_instance` to avoid a traitlets error.  Otherwise, this is great!  Please update this line and I'll merge.